examples/oapp/package.json

```diff
@@ -43,6 +43,7 @@
     "@types/mocha": "^10.0.6",
     "@types/node": "~18.18.14",
     "chai": "^4.4.1",
+    "concurrently": "~9.1.0",
```

Comment:
You didn't update the compile on this file?

Reply:
Done